docs/bigcommerce-for-wordpress/setup/multi-site.md

```diff
@@ -5,32 +5,30 @@
 <div class="otp" id="no-index">
 
 ### On This Page
-- [Getting Your API Credentials](#getting-your-api-credentials)
-- [Setting up a WordPress Site Using API Account Credentials](#setting-up-a-wordpress-site-using-api-account-credentials)
-- [Additional Resources](#additional-resources)
+- [Getting your API credentials](#getting-your-api-credentials)
+- [Setting up a WordPress site using API account credentials](#setting-up-a-wordpress-site-using-api-account-credentials)
+- [Additional resources](#additional-resources)
 
 </div>
 
 When connecting more than one WordPress site to your BigCommerce store, you need to use an API account to link them. If you try to connect using the 'connect your store' flow, which uses a BigCommerce app to streamline the connection, your first WordPress site will lose its connection to BigCommerce.
 
-Multiple sites can share the same API credentials or you can choose to create a new set of credentials for each site.
+### Multi-site and subdirectories
+Multiple sites can share the same API credentials, or you can choose to create a new set of credentials for each site.
 
-<div class="HubBlock--callout">
-<div class="CalloutBlock--">
-<div class="HubBlock-content">
-
-<!-- theme:  -->
-
->### Multi-site and subdirectories
-> Using the same domain on more than one channel is not supported. The Channel Site API does not currently accept paths on a domain as the site’s URL.
+| **Configuration Method** | **Is Supported** |

```

Comment:
@hatertron3000 Hey again Patrick --here's that table we discussed. Does this look alright?

Reply:
The table itself looks good. The branding is off in the link to the docs below (BigCommerce, not Big Commerce)

Reply:
Stellar catch! I'll get that cleaned up and then merged.